### PR TITLE
Using tf.contrib.losses.softmax_cross_entropy

### PR DESCRIPTION
**Pull request by alvarojoao**
tf.contrib.losses.softmax_cross_entropy avoid NA values during the training basically computes the cross entropy loss more carefully

### CODE REVIEW

**Comment by blackecho** [comment]
Thanks! :)
